openshift-charts-repo.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: helm.openshift.io/v1beta1
+kind: HelmChartRepository
+metadata:
+  name: openshift-helm-charts
+spec:
```

Comment:
Please add human readable name under `spec.name`,  similar to what we deliver within Openshift payload: https://github.com/openshift/console-operator/blob/master/manifests/01-helm.yaml

Reply:
Got it